fe1-web/src/features/lao/functions/lao.ts

```diff
@@ -28,12 +28,6 @@ export const getCurrentLaoId = () => selectCurrentLaoId(getStore().getState());
  * @param laoId The id of the lao whose channel should be returned
  * @returns The channel related to the passed lao id or undefined it the lao id is invalid
  */
-export function getLaoChannel(laoId: string): Channel | undefined {
-  try {
-    const h = new Hash(laoId);
-    return channelFromIds(h);
-  } catch (err) {
-    console.error(`Cannot connect to LAO '${laoId}' as it is an invalid LAO ID`, err);
-  }
-  return undefined;
+export function getLaoChannel(laoId: string): Channel {
```

Comment:
here and elsewhere, return types have been changed but not the documentation

Reply:
👍

The reason for changing the return type is that `channelFromIds` does not ever throw 😅

Reply:
Wait ! `channelFromIds` doesn't throw, but the `Hash` constructor (actually, the `Base64UrlData` constructor) can, if the data is not a valid base64url data !

Reply:
Oh.. 😵😂